puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/tarballsRelease.groovy

```diff
@@ -87,8 +87,8 @@ void build_tarball(project, version, ruby_ver) {
         withRVM(["${rake} pkg:generate_source"], ruby_ver, project)
 
         sshagent(['deploy-downloads']) {
-            sh "ssh root@web02.rackspace.theforeman.org \"mkdir -p ${base_dir}/\" || true"
-            sh "rsync -v --ignore-existing pkg/* root@web02.rackspace.theforeman.org:${base_dir}/"
+            sh "ssh downloads@web02.rackspace.theforeman.org \"mkdir -p ${base_dir}/\" || true"
```

Comment:
what is this `mkdir` for? `rsync` will happily create it for us if it doesn't exist.

Reply:
Perhaps in the past it didn't? I'd be fine with removing it and seeing when it breaks. We'll probably see that in 2.3 branching when we get to it.

Reply:
Perhaps in the past it didn't? I'd be fine with removing it and seeing when it breaks. We'll probably see that in 2.3 branching when we get to it.